README.md

```diff
@@ -141,6 +141,42 @@ The `style` and `script` preprocessors will run *after* the `markup` preprocesso
 <img src="https://cdn.worldvectorlogo.com/logos/browserstack.svg" height="80" width="80" align="left">
 <p>To keep Svelte's performance in check, we use BrowserStack to quickly run benchmarks for each PR that immediately give feedback to the contributor. You can see how we use BrowserStack in the <a href="https://github.com/sveltejs/svelte-bench">svelte-bench</a> project and check out BrowserStack's services on their <a href="https://www.browserstack.com/">website</a>.</p>
 
+## Development
+
+PRs are encouraged and always welcome. Pick an issue and help us out!
```

Comment:
"Pull request" might be more noob friendly than "PR"?

Reply:
Done